monitoring/prometheus/prometheus.yml

```diff
@@ -0,0 +1,43 @@
+global:
```

Comment:
I think you need to remove some comments as you did in datasource.yaml to make the file cleanner.